jsonnet/thanos/config.libsonnet

```diff
@@ -0,0 +1,234 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
```

Comment:
We are trying to remove usage of this library. Take a look at what was done in kube-prometheus to remove the dependency on ksonnet, for example in this file: https://github.com/prometheus-operator/kube-prometheus/blob/master/jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet